### ISSUE TITLE
create translation base text file and mechanism

### ISSUE THREAD

**Issue by olga-jane** [olga-jane/prizm#1055]
- [ ] structure for localization: 
- folder Languages near the binary, create on installation (already done)
- folder Languages/Resources, hidden, create on installation.
- base text file should be created on installation (call newly installed application with "template" parameter, or provide a way to generate the base text file after compilation and pack/unpack it on installation).
- additionally, should be created (or extracted from package) file run.bat to the folder Languages.
Content of run.bat:
"for each file Strings.??-??.txt in current folder, start resgen.exe for it, and save the result to hidden folder Resources."

- [ ] code changes:
create new constant Directories.LanguagesResourcesFolderName, which should have value Languages combined with Resources

- [x] base text file
I added some code in Program.cs. Your part is on //TODO on processing "template" argument.

**Comment by olga-jane** [comment]
ps. most of forms names could be taken from appropriate unit test case (one form is currently missing there, see "Add new parameter to non-required mill inspection operation #1027")

**Comment by olga-jane** [comment]
please implement first 2 checkboxes

**Comment by olga-jane** [comment]
Additional changes required: run.bat should do something if resource file already exists. I.e. 
1) check if file exists
2) check if it could be renamed. If could, rename to <old_filename>.datetime
if couldn't (apparently Application is running), give mesage that Application should be closed, and exit script.
3) only after (2) success, resgen should start the work.
